src/platform/forms-system/src/js/components/FormNav.jsx

```diff
@@ -107,41 +107,19 @@ export default function FormNav(props) {
         setIndex(index - 1);
       }
 
-      return () => {
-        // Check main toggle to enable custom focus; the unmounting of the page
-        // before the review & submit page may cause the customScrollAndFocus
-        // function to be called inadvertently
-        if (
-          !(
-            page.chapterKey === 'review' ||
-            window.location.pathname.endsWith('review-and-submit')
-          )
-        ) {
-          if (
-            formConfig.useCustomScrollAndFocus &&
-            (page.scrollAndFocusTarget || formConfig.scrollAndFocusTarget)
-          ) {
-            customScrollAndFocus(
-              page.scrollAndFocusTarget || formConfig.scrollAndFocusTarget,
-              index,
-            );
-          } else {
-            focusByOrder([defaultFocusSelector, 'h2']);
-          }
-        } else {
-          // h2 fallback for confirmation page
-          focusByOrder([defaultFocusSelector, 'h2']);
-        }
-      };
```

Comment:
Removed this "unmount" callback because it was calling the `customScrollAndFocus` two times based on the previous page's content. Also, it's only needed for the review & submit page; FormPage does the scroll and focus work for pages within a form